earth_enterprise/SConstruct

```diff
@@ -24,6 +26,27 @@ from khEnvironment import khEnvironment
 
 EnsureSConsVersion(0, 96, 92)
 
+# Check for the presents of the passed argument on
+# the command line.  If exists return a formated
+# string that can be passed to gradle else return
+# an empty string
+def check_for_argument( argument ) :
+  result = ""
+  if int( ARGUMENTS.get( argument, '0' ) ) > 0 :
+    result = "-P" + argument
+  return result
+
+# Check for command line arguments passed that must be
+# passed on to gradle
+def gradle_options() :
+  options = ""
+  options = check_for_argument( 'debug' )
```

Comment:
I'm not sure how you expect this to work.  There's currently no well-defined `debug` for the `scons` build, and, in fact, `src/SConstruct` would refuse to run with such an unrecognized option.

So, typically, there would never be a `debug` option passed to `scons`, and this line would have no effect.